DQM/DTMonitorClient/src/DTBlockedROChannelsTest.cc

```diff
@@ -59,8 +59,9 @@ void DTBlockedROChannelsTest::fillChamberMap( DQMStore::IGetter & igetter, const
   // get the RO mapping
   context.get<DTReadOutMappingRcd>().get(mapping);
 
-  // fill the map of the robs per chamber
-  for(int dduId = FEDNumbering::MINDTFEDID; dduId<=FEDNumbering::MAXDTFEDID; ++dduId) { //loop over DDUs
+  // fill the map of the robs per chamber 
+  // //FIXME: monitoring only real used FEDs
+  for(int dduId = FEDNumbering::MINDTFEDID; dduId<=774; ++dduId) { //loop over DDUs
```

Comment:
hi @davidlange6 , DT has booked 10 FEDs, from MINDTFEDID to MAXDTFEDID, however only the first 5 are actually used. For historical reasons DT DQM was monitoring all of them, resulting in 5 empty plots and folders which disturbed shifter's life.
Setting MAXDTFEDID implies to modify DataFormats/FEDRawData package, so I found this solution easier since it only involves DT DQM package and the FEDs we really want to monitor.
I'm open to any other solution you suggest.
Thanks